src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -122,7 +124,49 @@ private List<QdmValueSet.Concept> getValueSetConcepts(
   }
 
   public List<CodeSystem> getAllCodeSystems() {
-    return codeSystemRepository.findAll();
+    // remove items that are marked as not present in vsac to cut expense
+    List<CodeSystemEntry> codeSystemEntries = mappingService.getCodeSystemEntries().stream().filter(codeSystemEntry -> !codeSystemEntry.getOid().contains("NOT.IN.VSAC")).toList();
```

Comment:
The mapping is only needed for QDM measures. We shouldn't be limiting the results from the DB based on the mapping doc for FHIR based measures.